osu.Game/Beatmaps/Drawables/UpdateableBeatmapSetCover.cs

```diff
@@ -41,6 +41,10 @@ public UpdateableBeatmapSetCover(BeatmapSetCoverType coverType = BeatmapSetCover
         protected override DelayedLoadWrapper CreateDelayedLoadWrapper(Func<Drawable> createContentFunc, double timeBeforeLoad)
             => new DelayedLoadUnloadWrapper(createContentFunc, timeBeforeLoad);
 
+        // by default, ModelBackedDrawable hides the old drawable only after the new one has been fully loaded.
+        // this can lead to weird appearance if the cover is not fully opaque, so fade out as soon as a new load is requested in this particular case.
+        protected override void OnLoadStarted() => ApplyHideTransforms(DisplayedDrawable);
```

Comment:
This change is going to apply everywhere rather than just the place where opacity was an issue. Is that intentional?

Reply:
Given that the old code used to work the same way this does (faded out immediately after a `BeatmapSet` set), I think so?

https://github.com/ppy/osu/blob/42c82cea6842b63fb8ce926c081537c4e5214daa/osu.Game/Beatmaps/Drawables/UpdateableBeatmapSetCover.cs#L62-L83

Reply:
I mean I *could* disable that behaviour if the cover is opaque, but that seems a bit hacky and weird. Not sure.

Reply:
Could make a private subclass of this in that control and let it change the behaviour there instead, since that's where the cover is not opaque if I understand this correctly.